pkgs/development/tools/glslviewer/default.nix

```diff
@@ -1,53 +1,41 @@
-{ lib, stdenv, fetchFromGitHub, glfw, pkg-config, libXrandr, libXdamage
-, libXext, libXrender, libXinerama, libXcursor, libXxf86vm, libXi
-, libX11, libGLU, python3Packages, ensureNewerSourcesForZipFilesHook
-, Cocoa
+{
+    withFFMPG ? true,
```

Comment:
Why do we want to build it without ffmpeg?

Reply:
ffmpeg is not a hard requirement, so it should be optional IMHO, I made it true as default because it's desirable most of the time.